flink-end-to-end-tests/test-scripts/test_sql_client_kafka010.sh

```diff
@@ -19,4 +19,4 @@
 
 set -Eeuo pipefail
 
-source "$(dirname "$0")"/test_sql_client_kafka_common.sh 0.10 0.10.2.0 3.2.0 3.2 "kafka-0.10"
+source "$(dirname "$0")"/test_sql_client_kafka_common.sh 0.10 0.10.2.0 3.2.0 3.2 "kafka-0.10" "0.10"
```

Comment:
This starts getting ridiculous:
- `0.10`
- `0.10.2.0`
- `"0.10"`
- `"kafka-0.10"`
or
- `3.2`
- `3.2.0`

🙄